cranelift/interpreter/src/state.rs

```diff
@@ -77,6 +90,33 @@ pub trait State<'a, V> {
     fn set_pinned_reg(&mut self, v: V);
 }
 
+pub enum InterperterFunctionRef<'a> {
```

Comment:
Minor typo, but this type name appears a bunch of places in this PR:
```suggestion
pub enum InterpreterFunctionRef<'a> {
```